shortfin/python/shortfin_apps/llm/components/kvcache/page_pool.py

```diff
@@ -115,6 +115,21 @@ def free_pages(self, pages: list[PageInfo]):
         with self._lock:
             self.available_pages.extend(pages)
 
+    def copy_pages(self, src_pages: List[PageInfo]) -> List[PageInfo]:
```

Comment:
Let's maybe call this temp_copy_pages and add a docstring / comment to mark this as a temporary measure & not performant.